scripts/perftune.py

```diff
@@ -643,6 +643,77 @@ def __intel_irq_to_queue_idx(self, irq):
         else:
             return sys.maxsize
 
+    def __mlx_irq_to_queue_idx(self, irq):
+        """
+        Return the HW queue index for a given IRQ for Mellanox NICs in order to sort the IRQs' list by this index.
+
+        Mellanox NICs have the IRQ which name looks like
+        this:
+        mlx5_comp23
+             <bla-bla>mlxX_comp<index>
+        or this:
+        mlx4-6
+             <bla-bla>mlxX-<index>
+
+        :param irq: IRQ number
+        :return: HW queue index for Mellanox NICs and 0 for all other NICs
+        """
+        mlx_fp_irq_re = re.compile("mlx\d_comp(\d+)|mlx\d\-(\d+)")
```

Comment:
Why `mlx\d_comp` ? It should be `mlx5_comp`, no?
The same about `mlx4` pattern.